### DISCUSSION TITLE
Oracle connection

### DISCUSSION THREAD

**Discussion by karanarora07** [iterate-ch/cyberduck#15421] [Q&A]
I have a chicago based storage of oracle. But I cannot connect it please give me any solution.

Thanks

**Comment by dkocher** [comment]
In general, please refer to the [documentation](https://docs.cyberduck.io/protocols/s3/oraclecloud/). I have opened [^1] to add a connection profile for the region which is currently missing.

[^1]: iterate-ch/profiles#72
